### ISSUE TITLE
Use fizz migrations to generate SQL migrations

### ISSUE THREAD

**Issue by aeneasr** [ory/keto#448]
**Describe the bug**

The SQL migrations are currently handwritten and ignore the fizz templating pipeline we've built. The pipeline should be used instead.

**Comment by aeneasr** [comment]
@zepatrik ping me when you work on this so we can take a look at the templating for the column names.

**Comment by aeneasr** [comment]
You resolved this, right?

**Comment by zepatrik** [comment]
Nope, this is still up. We will anyways have to do some manual post-processing, is it really worth adding fizz? E.g. Cockroach indexes can use the prefix, so we might remove some indexes there:

> Queries can benefit from an index even if they only filter a prefix of its columns. For example, if you create an index of columns (A, B, C), queries filtering (A) or (A, B) can still use the index. However, queries that do not filter (A) will not benefit from the index.
>
> This feature also lets you avoid using single-column indexes. Instead, use the column as the first column in a multiple-column index, which is useful to more queries.
>
> https://www.cockroachlabs.com/docs/v20.1/indexes.html

**Comment by zepatrik** [comment]
And we have to replace the table name with the template stuff

**Comment by aeneasr** [comment]
Ok, it should still work with other things for fizz templating. Maybe we allow our small CLI script to support raw SQL templates too and then it just copies them over without changing anything?

**Comment by aeneasr** [comment]
Moving this to next milestone, but it needs to be addressed!
